odoo/tools/translate.py

```diff
@@ -319,6 +319,14 @@ def html_translate(callback, value):
     return value
 
 
+def get_text_content(term):
+    """ Return the textual content of the given term. """
+    return html.fromstring(term).text_content()
+
+xml_translate.get_text_content = get_text_content
+html_translate.get_text_content = get_text_content
```

Comment:
This is a _proposal_ to make `get_text_content` specific to those kinds of translations.  What do you think?
It also opens the door for other kinds of translations to provide their own version.

Reply:
Seems to be a good idea indeed. 

You sure have a better understanding than me of the low level Odoo architecture. 
So I trust your idea is a more flexible way to handle this change.